packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -44,6 +46,21 @@ export interface DesignSystem {
      */
     designUnit: number;
 
+    /**
+     * The number of designUnits in the default height at the base density (0).
+     */
+    defaultHeightMultiplier: number;
+
+    /**
+     * The number of designUnits used for horizontal padding at the base density (0).
+     */
+    defaultPaddingMultiplier: number;
+
+    /**
+     * The type ramp size to use at the base density (0).
+     */
+    defaultTypeRampSize: number;

```

Comment:
It's here so we can adjust it by adding or subtracting for density. Since the type ramps are keys, that sounds harder to deal with, unless perhaps in the type ramp they should just be numbers.